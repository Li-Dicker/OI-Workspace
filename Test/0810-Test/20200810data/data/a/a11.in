10
196
80270 13930 61423 59311 82534 75016 34745 43699 47096 84605 84695 33423 27787 26927 86945 93251 8381 79122 53086 64756 51795 74860 95076 83494 5877 68365 77437 24160 31382 9568 12590 78970 9228 94441 59059 78235 45586 40852 90528 96233 23444 62990 64433 65723 9166 32770 73165 9626 22376 91841 84736 36035 7787 76622 69373 14196 52827 34389 13927 79419 5474 62292 60053 43263 81537 561 51123 73893 45356 4999 28267 91796 87467 55859 75330 49874 31091 40224 12989 68470 3736 13640 8308 94240 12417 26844 27405 15657 56175 87162 56533 57804 7813 22803 71429 68227 1861 90316 20312 59433 27856 89930 6440 37405 93725 3883 87787 19514 92818 16584 54281 93892 63660 45188 47465 96843 21578 85604 35829 70287 45912 85669 18315 37280 14764 6117 75084 58443 86220 36424 31248 97145 57160 57172 68829 21971 19912 273 55830 21277 44440 88434 95760 67584 65851 69098 42239 79881 4158 27665 48301 71849 2853 60704 92568 39909 53534 9170 72859 31524 72061 29678 1077 65726 75903 29198 6677 54191 17214 92440 15397 26610 21378 28124 50840 80891 85361 82660 85206 842 81585 44191 98728 4433 94817 69572 92338 43653 97125 48657 22510 19762 80444 42813 27190 30845 
13405 16781 40225 3882 45844 21585 26287 14321 19584 33436 45083 20294 7333 29141 49425 20758 19131 7652 4850 23104 28242 7857 24110 16673 29148 7404 39642 21420 43453 48782 21522 12401 4073 48194 36527 9840 30770 16719 23289 38497 7095 33384 16174 29426 12865 26696 39784 4119 42501 39398 47976 43588 35682 19944 31478 17295 30253 37728 15544 23613 17312 13442 47912 37729 24514 19138 33315 14169 40793 24822 9773 19875 30285 34000 13407 2959 41658 40276 39935 12633 40574 32576 33925 11064 32131 27275 30322 12374 1625 41300 19935 34699 2465 41201 13326 5795 40789 21383 27349 43429 31632 6992 15926 34000 4876 17362 440 38838 33558 2640 1773 20088 34924 34467 11857 43491 42934 19878 14272 21917 11384 19550 48519 31596 3573 24815 20359 19039 12748 48368 45348 16273 15213 19769 9538 29854 12107 32811 22257 38147 23574 1063 9660 49969 40916 255 3001 24334 18299 20854 31468 22190 16629 8676 40791 9854 14274 886 46466 40923 40922 17023 49607 42029 28606 5601 45962 29866 16283 45193 45632 33293 46741 1071 3075 36784 9774 48940 18932 41900 41505 35764 5188 2129 15128 27340 25222 26044 40646 32523 44238 11465 5510 48743 46494 41049 
200
22634 91144 43783 54539 20362 77522 64775 75465 63269 40107 38782 67586 18563 25901 75980 9552 21758 45995 24270 69619 84361 84347 89079 16942 6524 37659 3545 31938 96294 70186 32170 25803 18208 74333 1245 50616 60205 31260 86375 32119 13086 24951 18290 29195 59109 43777 35713 56311 54590 23042 16209 99140 16578 93262 56514 50233 64926 54494 75458 50452 41552 93346 93301 1732 85027 84737 46977 97454 81609 7944 96156 7210 26142 5262 5468 95170 55235 75121 29381 33274 88637 55812 38764 63911 26044 99835 36588 76760 15222 48207 2797 5652 71216 21429 47639 35568 87127 85148 73133 66448 81744 56038 81580 78780 67211 47946 98251 35947 4835 75856 83010 31014 94433 51437 59520 38481 55953 59030 30021 82756 33729 61102 86655 45101 30881 49645 36150 55454 68776 89216 47071 35286 43073 12675 86029 21969 86338 6879 33629 7284 32296 47707 9595 22018 26141 98979 52333 13778 91907 50642 52225 82649 11472 24179 75576 67201 80857 63565 42199 47146 30158 33704 28180 39237 29987 29684 65880 19877 84425 85097 9553 46850 2742 48280 38544 27946 57658 72804 19235 60974 6487 228 558 2180 24768 46975 6778 12756 1185 20555 37835 37124 79395 28884 51034 29942 90499 4687 64495 8140 
18079 29050 7806 29890 478 45955 8640 38928 38695 15212 14617 48124 22531 37376 3423 27822 14266 20061 20126 44033 10169 28935 23586 20019 25272 7158 34598 36524 32157 1905 3022 13357 42222 3767 11766 46484 47961 31566 27383 5517 41130 33509 45390 32882 49494 49562 32322 23345 24620 45733 26510 8020 49723 40173 4576 39741 24985 9978 37167 32285 37151 977 16050 12275 31756 31839 13617 49741 47201 15932 1425 1460 16245 36431 40046 14280 825 36105 34415 8042 8030 43170 18248 25172 7294 2754 44555 12933 8121 40652 39345 11191 4504 28348 30756 5489 18884 43543 3460 44068 13128 30404 20551 24635 41398 23912 35209 16385 14562 7644 42892 33916 16329 30928 33024 36475 23678 45891 48116 49173 37199 10863 14336 12170 2599 42567 45191 47449 14066 47081 7257 32150 48281 401 32448 35310 35988 32079 45482 5685 8415 45474 14430 12623 32720 22724 8543 12542 23825 233 6043 20474 2663 49500 49488 10291 39124 457 18426 6690 20134 35150 995 49463 26889 8974 28596 11015 27371 7799 46489 26598 42138 15047 35336 25819 26266 27482 35606 8478 23691 38809 43281 39529 2919 48224 29053 44484 2810 13688 25048 11079 8268 19068 48217 12805 48373 12611 1869 32370 
196
46618 12295 96136 80126 89308 78937 37048 13389 5218 87140 45430 92542 85534 29331 46716 98857 47418 47485 49412 95511 42374 31298 45670 44496 33098 13547 40762 3733 63788 52135 51718 58917 40114 80601 89591 82566 52308 75752 71321 6811 22125 12958 58871 4158 42553 50411 22209 65091 22831 16967 57200 35672 74214 40731 51734 28679 4381 81255 85798 62637 1238 61226 92280 60881 66649 46389 9820 60263 16865 11456 20748 46118 65596 3891 98608 12748 40152 16290 64393 36963 29097 79362 83513 64342 4874 59375 78864 59888 62790 19750 73817 36936 37615 59538 37118 47085 44020 50415 31408 27426 32443 43806 84857 80091 6651 93461 25248 47936 49962 70752 1716 56022 30018 18100 45594 31617 54183 13658 2010 58939 85879 39682 49296 594 83833 36876 47229 98362 13123 58264 12562 70055 63210 43555 17403 80631 37781 31593 15794 86806 58326 41183 67208 81889 730 72484 45978 69536 65102 90969 15880 40392 12142 61331 59590 7806 99963 20388 8402 48061 21145 12386 93419 33429 82975 66779 52361 1579 67850 15680 37611 95127 93989 36903 20388 83822 73951 27623 32996 31558 88549 81589 67718 99054 51902 28110 9647 61927 96921 36008 16062 9784 43464 78369 35548 41392 
26764 1144 460 49551 15859 919 27426 32476 10369 49529 13013 15423 25749 33813 30514 42911 14780 13887 13303 17768 35490 13958 6267 20524 29183 45003 25808 23199 26978 41148 33408 20067 17484 15810 28343 12544 42704 35264 29539 12973 113 45549 20405 231 7915 23547 13296 29400 45019 2893 28995 43882 24966 44726 27689 48561 24528 47337 6608 26061 47226 25230 36339 48088 35439 35334 2870 12505 27305 2259 38704 7070 47191 23118 20132 44646 24623 46927 10369 49175 34037 48769 47904 3960 12702 8760 4432 23407 17035 8720 40113 22066 22570 47347 16347 2611 23036 46119 46150 43095 7936 13801 37009 9083 15391 36905 41898 10591 27647 40345 3486 6935 31175 26146 2460 44062 39088 14168 38265 41778 49475 25221 30138 31773 4064 25986 20328 17268 45845 37661 16476 12066 42918 5275 8266 22658 43365 903 33979 5487 35967 2967 44723 16989 13697 4917 6145 46103 19652 23566 19876 31478 8075 21160 7057 42945 13084 37545 46242 42947 41457 848 12489 37916 23637 41695 22146 20064 33902 27715 5730 41261 24775 38422 18949 43340 30943 45600 8623 15440 38550 27828 1034 14058 21491 5018 9152 13720 5216 31861 22050 41557 19209 48644 1784 3562 
196
18760 95540 50018 91151 97941 15788 67198 36795 82554 56641 29445 30751 77857 10932 54788 19851 95565 90752 53472 58555 36844 42413 22617 36080 79411 1691 48396 33688 86328 23774 84583 49948 80329 39919 87233 40151 62479 7384 11548 10232 84592 29753 49713 66116 18969 51155 27498 29272 6231 19260 70515 68986 19760 23853 90490 80793 40277 35202 66443 87433 52829 9668 47592 51498 90005 41323 58635 98045 52331 42947 58767 59832 13418 15973 50130 73614 56940 16096 13938 74802 16036 84376 23712 96887 90779 50459 56870 5368 66317 65131 55642 17195 94024 23532 72875 60441 55644 28549 42870 46573 29937 60059 4245 97076 17132 15428 84036 99563 54931 81890 13598 4804 78873 57539 36056 89309 87751 95720 28723 63951 81056 1482 6921 80851 86344 49691 2567 38413 9928 22438 2196 75435 77493 78234 32044 87700 98049 20271 79943 6669 95857 42755 47403 89097 66688 38380 45215 49204 68299 95781 27694 63222 29452 42927 62063 14854 88115 18271 31325 60034 93480 91292 82477 33602 90204 35056 72003 41505 68119 71835 23973 48007 20225 68082 14843 10508 95534 36 36334 33727 68595 56898 62430 22878 29975 71741 46441 9865 11897 13599 35747 49112 33922 50893 78738 76310 
13538 23380 49389 4065 31176 11499 29364 14698 30921 11779 34035 1315 29200 23234 32724 13579 21954 30946 6709 2784 41132 24518 4515 24188 22441 48619 49784 43460 23543 29457 17335 5745 39425 40136 1316 25304 12552 19132 41961 24303 49755 29835 7175 27865 629 4561 18633 17721 28800 44117 33728 40707 40465 20669 25755 48668 13137 32544 16897 28608 13566 4731 7403 30547 16785 2255 37220 15645 32659 3422 27530 573 36019 26644 11349 37688 47914 39933 5791 33556 9699 9096 21597 29880 44864 45797 35812 30558 48312 49929 46390 7091 18902 39445 28334 4350 2833 22266 10489 48327 37643 43812 9468 2590 27914 6441 32354 544 5841 14610 43767 5688 30443 3222 33705 19354 33941 36550 35311 43752 18558 29170 24754 24531 43966 24370 44559 28970 36104 15786 14859 35842 34675 24528 34611 47344 26080 17301 4260 23804 26407 43862 38383 33515 14776 47628 11696 6836 3145 43391 31416 29403 15915 14502 16741 23457 14916 28588 8064 2799 45856 24994 38186 35895 29204 4661 40468 47030 48369 44806 8053 28923 25747 5065 9558 44407 30190 13467 6540 27260 30670 40550 27868 44173 5813 39140 28863 26025 15818 11524 12929 10422 15809 18624 41435 49982 
199
93035 17525 5487 84877 27441 47429 75638 96742 41187 5510 10482 75842 10916 96104 93968 24067 82431 5991 52556 57518 34887 17983 12071 12225 75669 89528 8580 52824 14479 77912 37577 83095 94114 18584 4129 43519 46726 4813 71802 18227 58806 72561 26060 68471 9628 47079 31413 94015 37657 37697 80320 33581 31786 61509 5542 79045 59662 18645 14590 37785 72163 55232 11929 20612 77776 72899 5522 29200 10674 1602 38868 38682 36886 55154 4060 88246 1928 53458 85415 23244 93084 69349 75646 85521 29696 70840 27678 55290 99194 71081 94209 61493 22793 13265 31336 97395 58524 99397 56401 61747 19687 12750 7790 33961 49927 89330 85663 11628 10386 76217 35123 85455 69713 64213 60682 75808 35517 32485 14247 30074 9726 91068 24535 41377 97140 17807 17574 85200 10139 64465 29994 80760 87180 67428 22517 8111 87124 60285 92294 62549 1676 2843 46657 67339 51200 95194 13489 85008 5324 47667 48482 13943 19741 73680 38556 44011 33907 28379 82805 83977 86631 50901 2425 7750 75726 73353 46768 73135 60803 9098 92860 89747 26078 25384 33617 70287 99754 22178 26809 80653 8192 71807 71211 30471 51439 35427 83966 21271 31096 50301 79892 6689 25634 94006 48620 80621 34887 28611 53899 
30564 15313 16497 40976 3002 39740 39396 43302 14076 14546 29314 18532 27845 34800 26849 23819 15832 30825 49436 40424 5475 41336 32828 12285 25542 5545 17563 47847 16276 35948 26745 29060 19042 44987 28418 29684 28104 43981 3508 8448 32763 49042 11502 38726 31869 25329 12760 9995 30382 14575 14872 32332 142 380 46478 611 18545 40975 33109 1442 13706 23071 48671 13190 17673 31206 47556 34666 1567 3991 23614 7947 38791 47964 16430 70 29871 21071 30343 19705 10786 3731 4111 49922 24733 29944 44755 1678 26988 40071 5893 71 40475 11882 6417 23366 42011 28960 47396 24404 33064 3419 37995 49937 17674 11152 37597 26896 6350 27057 20328 12120 27782 13330 27499 34421 35130 26788 38686 22902 10552 16114 23879 31178 45894 28457 38629 7036 18304 24330 17259 14236 40222 44539 44257 39413 12560 30538 20172 20090 44085 42107 11337 27787 2410 36337 7821 37750 34124 48700 3157 34142 44430 21034 23632 30810 12069 44361 22922 2328 19988 40959 43051 5413 25110 25113 49333 12467 24500 5958 26256 16401 27575 18133 36500 27574 47805 6717 24075 30785 24481 31719 20914 23885 13135 33647 25881 35045 21080 49786 15172 45973 43661 5577 43744 36742 11542 12949 18062 
197
37430 70007 21244 21871 99798 32147 29655 68174 36222 28281 34536 88644 3588 72257 92321 45482 13023 94456 40453 1970 95156 27441 5859 77867 89682 68954 55380 84588 10341 80825 81333 47604 13300 44997 17269 13666 85526 46283 7394 93214 7818 39946 59884 29503 13563 46048 27898 26573 30594 6637 3849 24894 64956 72401 70793 96290 60145 2515 13302 72378 64741 78744 65260 39258 44361 23622 9630 72433 71761 53821 96627 56577 96601 80501 41564 44551 834 91838 63000 85352 12836 32167 52671 75428 41547 97771 87334 76272 23194 24192 79189 80667 44295 88559 87512 12434 83076 29408 10010 89098 14695 34555 81680 3775 93596 44785 28126 96042 97985 95198 12093 38642 20407 35536 70653 51013 25482 22610 46324 99231 15861 40399 18204 14355 17491 67875 44709 26619 37807 33343 15382 3745 21380 48508 92678 95017 75117 75118 53423 97074 76639 16872 17822 80158 68187 66522 64660 70525 66439 82370 22214 35251 19295 3330 3486 89679 1976 85648 44807 50937 26570 32926 6359 76217 46326 99804 77802 74116 1105 36950 7210 11583 36611 73520 65235 95638 16833 89615 94080 98636 28822 58284 69702 38025 99935 79877 98521 16594 69617 33949 55020 79322 87135 26853 79916 46611 67179 
24947 11811 41774 23754 2189 25010 4808 24444 3095 47082 3632 38128 16291 37125 15981 22066 31931 6731 40052 1644 24029 21560 38108 34366 48863 21294 46392 6305 10485 15953 31932 10918 16343 6884 268 22722 28229 33485 36058 45791 43201 10657 19144 10172 5144 32844 20296 19335 6403 24259 1732 22777 39327 41075 41179 41841 339 37975 28271 18465 39070 20709 21323 16808 21000 4559 12694 13122 47691 40867 49050 43911 45242 35734 826 22655 11798 47595 45275 1471 25639 31501 23165 48158 10608 37995 8360 30268 1141 40530 38360 32139 35254 13327 44801 28659 6095 20816 7612 28158 18104 19327 31629 44036 39871 23812 34812 8418 35492 28027 40869 33766 2695 6181 19819 49271 39869 27259 39247 16277 21900 32856 12066 24685 41373 39782 7830 3156 37084 17123 44959 17962 14211 4850 21668 38376 42182 13954 5838 7934 45840 22508 44589 42534 46124 4232 19561 36574 33944 28643 6870 10006 28405 15759 22131 18619 37038 28790 18968 3475 32513 1529 45690 20554 9077 19609 30478 23931 48643 12477 9060 19579 38963 19805 36038 43064 33099 12680 23697 23621 7244 8784 10823 13970 24054 43139 25732 5413 17697 44979 7095 15953 36118 8969 35277 47067 39860 
199
98648 93071 3045 22167 89990 73841 10622 74001 34026 70911 15356 63742 26009 24665 22412 40964 92357 69920 61311 2004 78377 538 327 69861 3609 64604 35124 65186 19915 14584 35720 89320 91838 66134 79465 41288 42690 53090 83489 15796 54891 80212 61837 46006 27644 35066 41778 93813 70245 24632 3044 94070 50393 24754 54445 41263 54482 24960 60291 1667 80248 96213 82342 15678 74166 27064 21168 61101 91067 42476 706 9884 18816 20474 47271 44933 98124 12406 13344 36827 1679 28276 7691 39444 26043 9759 84455 28596 67154 90965 17574 50426 42322 36763 6559 66566 57081 87395 11121 8993 7670 98244 45848 64511 18218 28780 11243 6705 63240 97273 39480 29815 71842 2278 31948 6463 51610 87227 75370 41547 85123 71752 53075 34769 45186 93441 46644 44431 73501 11213 17592 59820 78051 60259 41512 66453 9850 20776 78644 98957 99643 89530 2778 12812 20147 20604 26251 47107 62044 5202 64741 7855 99366 25613 34325 59917 39221 96956 96767 95826 67641 11750 61372 40985 79803 51997 71257 51664 23969 31867 16779 64508 67574 29750 548 9476 35156 37447 42237 96961 5307 98737 4798 40586 84482 28931 5742 49160 92864 71746 27947 88733 70723 87542 71206 96256 70782 47777 24928 
2649 32675 26322 30540 42221 46208 29961 19386 20 2402 33944 18470 22439 17299 24480 46306 42454 39565 71 28892 7153 6766 29399 5245 20622 42233 11758 9887 46720 44184 7134 48058 35653 16332 33730 34323 23348 3557 45714 2018 8063 16690 24934 479 4181 23645 577 3003 17821 23587 3760 10404 23376 29359 6869 13015 22916 4813 18030 6041 27583 26544 13310 2258 10415 37077 818 27943 5999 18 44253 22249 24384 38411 32562 4138 4827 2719 24488 17917 33850 49160 15860 22961 16911 46576 25485 14782 48582 15853 38628 4734 5214 10964 33967 9009 47821 22149 5731 29364 3832 46280 24727 14260 41251 24119 48749 9492 43885 39381 43327 48309 43469 33220 41746 43156 48997 35336 46142 6424 45396 27792 17868 44592 29795 30213 38964 3128 7729 28136 34520 342 10131 12026 15962 34253 19885 47300 30929 48262 22231 17426 8998 27996 35518 5103 43327 674 9127 11540 42126 21424 15979 7207 37348 4963 24622 48433 45019 22251 48610 5748 32558 16723 23020 9370 46757 25711 19787 26926 8608 20840 21793 26531 15502 31473 12925 33445 1143 33109 43008 36573 36750 40123 42576 26888 24152 2755 17830 32579 4759 41568 11668 25687 6247 46498 9249 47276 8353 
198
1866 48276 73536 72562 61690 3515 6312 84477 35505 86665 40046 23702 80197 67591 75784 15270 34177 65220 82114 43800 99201 36943 71756 35561 63122 74586 20130 17941 38566 80132 98125 92170 41832 86728 81943 77311 72537 99703 42035 76961 88887 98076 23319 85202 77930 50985 66630 38830 68440 91234 82911 60898 4257 8842 98771 81593 96495 89888 76696 1811 15166 35126 75541 75972 11753 33043 72202 31703 15668 50507 69792 16281 2832 14044 24161 62355 67568 80957 53504 77018 32127 90959 48738 45130 82217 79939 29131 28283 46167 74264 19664 17497 9945 47181 75360 63039 78522 52404 16446 71458 81475 50867 85216 79732 22304 71067 86022 29425 7436 20397 27702 75060 96948 96551 58227 2586 69489 87817 61590 56061 93008 3479 66233 47476 91121 90748 67667 3433 27099 49018 1504 79758 7808 13797 24646 52541 70826 44682 35988 37565 75648 17488 34737 22303 62816 27171 44244 29703 8746 86453 80224 91121 1502 62965 84436 56138 44986 92812 71704 60216 62219 55300 96149 62931 48915 34221 44210 72480 19387 33559 10766 66099 97761 72949 69263 63976 6500 18355 47595 3163 62430 46200 44263 82789 21431 61267 89787 24262 3614 24447 88645 22973 68844 8226 23351 35248 78791 54557 
27036 27159 24445 46250 45229 14109 42606 2619 31623 2790 19927 15507 39419 3025 46119 16645 33540 5820 45692 43142 44305 29718 39502 40835 1474 27269 31897 32051 32996 25564 10254 13422 11890 4035 3411 16132 40152 6081 49177 47482 48365 1654 32827 45222 318 31309 9001 26100 20182 27419 25398 20871 24405 33706 27810 4834 10580 24138 8264 2246 38351 6484 17314 33370 25806 21224 32902 27510 32457 28670 37736 26952 18801 38786 42022 21367 8217 39371 20837 17702 14795 20395 8185 14797 36998 32857 30304 2300 40948 2768 13116 30838 34821 9640 48996 15349 43196 45031 3551 15310 8958 40259 11320 5616 15285 3597 45717 5092 4103 13187 29487 18092 23557 28337 22966 38825 30313 45555 3836 42281 25255 7754 40541 15832 15789 27941 5556 8747 44679 9022 43553 28599 40630 13044 29177 46339 29126 44829 20045 10638 25824 2873 20419 42891 7765 34089 36214 22369 6001 9136 30398 36966 17926 1280 33233 45636 19799 48363 10743 32998 23318 9157 34240 8536 29819 27460 6759 29238 45093 11194 19447 852 44017 31990 38379 28091 42528 20189 31657 46567 32644 15451 10603 26475 34406 3107 2512 49326 46468 49977 17930 18556 31782 41417 6923 41773 47501 2583 
199
78888 1024 40477 65030 56718 13125 98482 68554 92582 59535 94893 76522 26191 98418 21179 30509 96456 29278 82932 3163 92210 22465 26360 83695 99833 29604 66785 39285 34475 48980 78746 92900 54661 80311 49549 20573 74004 94980 12038 32352 18440 69584 58357 10564 72183 31577 23835 7377 22162 28722 71106 24030 38211 88025 25057 70326 55118 93331 23953 72279 465 18006 95354 54045 17687 15825 84325 29409 38256 48299 73704 3702 76747 98971 86070 13762 66685 5540 39865 56553 18459 73928 99123 7928 54943 28725 73907 75395 69199 26995 40364 9106 78367 27062 39305 21625 89578 39877 98630 67635 51925 2623 13098 85341 11281 3851 85113 12093 98175 73698 86613 99548 82368 10442 97934 35774 53035 15069 14090 4755 97415 29419 59005 56216 2675 67124 73952 90898 6638 44033 55943 20983 30474 34936 9006 27446 14969 91229 8369 87432 907 64612 66696 41495 60088 64057 73660 44380 51089 68385 32585 87395 39205 46868 82142 29708 48612 80009 3383 78555 59393 24457 15164 1698 2382 43691 92731 67202 33273 48331 35214 96843 65056 95527 95764 74518 81410 9725 24471 11285 28663 59173 67194 29167 13765 12610 90606 68814 45615 56692 89256 14303 89206 2123 43136 31658 57085 74802 41356 
9194 24554 33600 33144 35209 18543 28693 38103 5516 2716 12565 17063 28155 380 24254 37978 1355 2779 30905 12374 1093 39688 48816 4382 8402 6150 24547 10772 34972 18240 9462 46177 17250 9473 18334 28254 15876 34146 17384 29524 34279 31964 38077 36373 152 30093 20611 19368 45468 24632 2263 24078 1677 8194 17433 29348 45677 19641 37285 11462 24267 12726 40562 22861 16296 25461 42883 1099 3383 38786 17484 43769 14108 5878 46329 36758 10636 38444 45269 27083 31424 14381 26897 42268 20329 13919 5736 28261 16622 18401 9321 39864 35679 21990 7222 46304 20122 13070 27596 16725 26037 36090 37278 37235 1946 22561 49404 6342 14113 47374 18594 6140 29945 10490 5503 15731 20520 25352 33756 37870 18158 15392 26904 37918 2690 45533 35323 12216 3262 25577 26987 30850 40401 25225 38193 19416 34687 45671 21828 27043 46461 35315 45446 45344 17961 24532 43623 8814 47093 45421 37033 21479 292 35969 43488 43052 8760 28016 47652 47685 49858 15951 34874 11915 24782 42585 30214 24664 3737 21606 22570 40102 40137 11792 33773 42826 17890 19806 27836 18609 13517 9849 15579 37884 33412 2394 29786 6835 34255 12 28018 21888 28207 20235 35421 40475 48164 46636 45117 
198
72095 57118 76882 14384 5324 35682 35722 52406 3511 68778 50411 97967 1472 29343 92642 4921 57519 26842 21072 35801 21594 88374 48168 25183 28444 29372 32035 75591 39492 47939 94309 18924 49254 69293 54674 44173 32050 38177 84638 79574 86000 34629 50256 71388 30802 75301 46370 13794 56595 13090 81988 48377 88681 34111 79849 60285 84103 9158 11158 39639 78558 29942 34468 16983 82732 20035 74732 43023 99543 10445 3136 89314 43873 40167 84864 75757 59589 60188 6603 10589 37309 6104 37577 94936 56501 90056 87907 400 8067 44677 10589 19678 5126 11428 50469 14460 20525 13710 9825 46555 95691 66839 67718 68452 65390 61088 29419 49110 2470 47053 85026 89611 16348 78938 34341 19292 12705 14899 50299 76718 24499 88716 17069 25794 83219 60265 63872 43244 63170 64320 18028 39562 18019 46558 72954 18529 82890 53315 13311 47558 35040 37955 11394 90944 7501 57149 71318 47204 5899 54341 5938 67869 80676 3714 66504 27351 26032 26231 48775 22210 88085 40082 39326 57666 87187 36244 57518 98665 48486 84109 84002 12514 76996 90885 99074 39980 29135 51619 98387 58634 58197 3219 72288 67195 2440 66810 11171 3350 59760 70438 23362 52178 99168 58376 64561 16748 66268 70022 
42512 18827 44029 19096 11971 22453 44525 25472 25880 45415 15611 12491 22462 20633 5455 11895 14927 47675 40014 48926 38552 23372 12616 36375 21571 43743 17946 34427 41977 3902 16068 19159 48065 24435 46756 24144 31742 19512 3841 47233 44635 3095 7901 1061 27228 8320 28462 18835 18072 10207 11860 19700 30223 15530 42120 37219 37958 47834 39803 24441 38426 47297 23249 38218 26645 12686 11908 42948 14038 36431 41342 25843 19978 48916 13848 13789 14448 10490 42781 35966 5847 17790 44467 46771 46711 13390 5036 12282 27030 10058 20209 41858 19999 25134 15932 28307 43217 25265 18559 16461 42431 41414 43657 22263 16512 13202 10462 5106 23567 46405 48849 22188 45611 31734 17598 24907 30216 49966 9044 42201 8902 32069 8923 46620 39537 35655 7669 49355 44065 44531 1986 22918 8400 11988 6794 14711 9862 42069 15430 32864 40891 5647 14032 46058 14045 2329 4815 5085 7381 23833 49429 12075 44364 49676 29142 47276 40275 16022 23036 42331 16199 7633 3679 156 26870 38077 17388 4844 49096 14421 46462 21842 49631 41945 38768 36825 31906 42587 0 33775 43827 35240 4220 32191 20759 23832 49273 24119 15751 33408 2167 34395 15258 48 24450 21925 46922 20672 
