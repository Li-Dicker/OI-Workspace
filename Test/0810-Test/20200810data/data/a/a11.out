Case #1: 14163354
Case #2: 13470421
Case #3: 13507531
Case #4: 14133251
Case #5: 13872468
Case #6: 14124332
Case #7: 13309342
Case #8: 14438806
Case #9: 14227550
Case #10: 13980214
